tests/test_levanter_hf_consistency.py

```diff
@@ -0,0 +1,104 @@
+import numpy as onp
+import transformers
+
+from jax.random import PRNGKey
+from transformers import AutoModelForCausalLM
+from transformers import GPT2LMHeadModel, GPT2Config
+
+import haliax as hax
+
+from haliax import Axis
+from haliax.partitioning import round_axis_for_partitioning
+from levanter.checkpoint import load_checkpoint
+from levanter.compat.hf_checkpoints import hf_backpack_config_to_levanter, hf_gpt2_config_to_levanter
+from levanter.config import TrainerConfig
+from levanter.models.backpack import BackpackConfig, BackpackLMHeadModel
+from levanter.models.gpt2 import Gpt2LMHeadModel
+from test_utils import skip_if_no_torch
+
+
+HF_BACKPACK = "ivanzhouyq/levanter-backpacks"
+LEVANTER_BACKPACK_CHECKPOINT = "gs://levanter-checkpoints/backpacks/backpack_170M_100k_steps_run_0424-new/step-100000/"
+HF_GPT2 = "ivanzhouyq/levanter-gpt2-small"
+LEVANTER_GPT2_CHECKPOINT = "gs://levanter-checkpoints/dev/clean-snowball-990-new/step-10000"
+
+
+@skip_if_no_torch
```

Comment:
let's also skip this test if the checkpoint isn't available b/c it's a non-crfm person running it

Reply:
actually better still would be a version that runs like the gpt2 test that tests hf == hf->levanter import == levanter->hf export

Reply:
will add